lib/transform-directives-for-browser.ts

```diff
@@ -40,6 +40,23 @@ function transformDirectivesForPreCsp1Firefox (directives: Directives, basePolic
     }
   }
 
+  // Rename `scriptSrcElem` values `unsafe-inline` and `unsafe-eval`
```

Comment:
Old versions of Firefox don't support `script-src-elem`. Why do we do this transformation?

Reply:
You are very right, apologies as it had been a long weekend for me.  I will remove.